### ISSUE TITLE
`bloc start` does not have a guide and will throw an error

### ISSUE THREAD

**Issue by PirosB3** [blockapps/bloc#92]
Hi all,

Currently working on `d05371434df030f298952f8b050de667c234759d`. Upon running `block start`, that states to start a local server in the documentation, an error is raised. I have attached the stacktrack:

```
root@default:~/bloc/lorem# bloc start
events.js:141
      throw er; // Unhandled 'error' event
      ^

Error: spawn ./node_modules/gulp/bin/gulp.js ENOENT
    at exports._errnoException (util.js:874:11)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:178:32)
    at onErrorNT (internal/child_process.js:344:16)
    at doNTCallback2 (node.js:441:9)
    at process._tickCallback (node.js:355:17)
    at Function.Module.runMain (module.js:469:11)
    at startup (node.js:136:18)
    at node.js:963:3
```

I have followed guide but still hang up on this error. Is it something you guys are aware of?

**Comment by kejace** [comment]
We're working on updating the Readme and the error message (#48). The actual error was fixed in #64 I think.

Make sure you pull the latest version from github. You might have to create your project again.

**Comment by kjameslubin** [comment]
@PirosB3 as I noted on gitter, you likely need to `npm install` in your new project. We are updating the documentation to reflect this.
